identifier/src/containers/Login/Login.jsx

```diff
@@ -67,6 +71,9 @@ function Login(props) {
 
   const { t } = useTranslation();
 
+  // to show password on click of visibility icon in password field
```

Comment:
Please use full sentences for comments. Means start with uppercase and end with a period.